spec/jobs/sufia_resque_queue_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'spec_helper'
+
+describe Sufia::Resque::Queue do
+  let (:subject) { Sufia::Resque::Queue.new "test_queue" }
+  let (:job) { double('job') }
+
+  context "no errors" do
+    it "queues the job" do
+      expect(::Resque).to receive(:enqueue_to).and_return(true)
+      subject.push(job)
+    end
+  end
+
+  context "timeout error" do
+    it "retries on timeout" do
+      expect(::Resque).to receive(:enqueue_to).and_raise(Redis::TimeoutError)
```

Comment:
@mjgiarlo   I was expecting the enqueue_to to be called twice once it would raise and error and the next time it would go through.  The test is that it gets called a second time.  So I would see the first call being an allow, but the second call is definitely and expect.

If you would like to rewrite be my guest.